lnwallet/channel.go

```diff
@@ -1313,17 +1316,65 @@ type LightningChannel struct {
 	// log is a channel-specific logging instance.
 	log btclog.Logger
 
+	// musigSessions holds the current musig2 pair session for the channel.
+	musigSessions *MusigPairSession
+
+	// pendingVerificationNonce is the initial verification nonce generated
+	// for musig2 channels when the state machine is intiated. Once we know
+	// the verification nonce of the remote party, then we can star tto use
+	// the channel as normal.
+	pendingVerificationNonce *musig2.Nonces
```

Comment:
looks like this is an alias for `localNonce`?

Reply:
Don't think I understand the question 100%, but this is used to sort of cache the nonce we create each time we verify a new commitment transaction. The revocation is sent right after with another method call, so this stores that until we need it.